src/bin/lttng/commands/enable_events.c

```diff
@@ -965,6 +965,30 @@ static int enable_events(char *session_name)
 		}
 	}
 
+	/*
+	 * Adding a filter to a probe, function or userspace-probe would be
+	 * denied by the kernel tracer as it's not supported at the moment. We
+	 * do an early check here to warn the user.
+	 */
+	if (opt_filter && opt_kernel) {
+		switch (opt_event_type) {
+		case LTTNG_EVENT_ALL:
+		case LTTNG_EVENT_TRACEPOINT:
+		case LTTNG_EVENT_SYSCALL:
+			break;
+		case LTTNG_EVENT_PROBE:
+		case LTTNG_EVENT_USERSPACE_PROBE:
+		case LTTNG_EVENT_FUNCTION:
+			ERR("Filter not supported for %s event type",
```

Comment:
I'd see this message as
`"Filter expressions are not supported for %s events"`

What do you think?